packages/gestalt/src/ExpandableModuleBase.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import Icon from './Icon.js';
+import TapArea from './TapArea.js';
+import Text from './Text.js';
+
+type Props = {|
+  title: string,
+  icon?: string,
+  iconAccessibilityLabel?: string,
+  summary?: Array<string>,
+  isCollapsed: boolean,
+  onModuleClicked?: boolean => void,
```

Comment:
@galaxy0101 can we make this required instead of optional?

Reply:
1) we will need an expandable card without summary in ads manager. 
2) we will need an expandable card without icon or iconAccessibilityLabel too
3) onModuleClicked can be made required, qq: why do we want to make onModuleClicked required?